grabl/analysis/DependencyAnalysis.kt

```diff
@@ -0,0 +1,46 @@
+package grabl.analysis
+
+import com.eclipsesource.json.Json
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.util.regex.Pattern
+
+
+fun main() {
+    val remotePattern = Pattern.compile("(remote) = \"https://github.com/(.*)\"(?:.*)");
```

Comment:
There's also `ssh` url.

Reply:
@lriuui0x0 addressed in https://github.com/graknlabs/dependencies/pull/210/commits/38fc9d8aa6ac2fb6b7fe21a82cb216b9867c41ca